src/page-building/landingpage/landingpage.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs'; 
+import { Career } from './career';
+import { Service } from './service'; import { Blog } from './blog'; import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+
+<Meta title="Pages/Landingpage" component={{ Image }} />
+
+# Landingpage
+The landingpage page consists of various sections you can see here
```

Comment:
The minimum story set of the landingpages. Yoy can now see the various sections from the usage perspective which is quite nice.

Reply:
Nice 👍 

Depending on the data, I can think of quite complex mock data we need to take care of.

Reply:
Yes, but we can place them in `mock.ts` files or just don't showcase them if not suited.